tests/common/mod.rs

```diff
@@ -126,3 +96,41 @@ where
         Ok(None) => Err("timeout waiting for snare to exit".into()),
     }
 }
+
+#[allow(dead_code)]
```

Comment:
I'm guessing this can't be `#[cfg(test)]` due to the way you test?

Reply:
Because each `.rs` file that `mod`s this file is a crate, it means that it can be seen as "unused" in one `test/*.rs` file and not another. It's a bit messy, but I don't know of another way of avoiding meaningless warnings.